tests/test_readDicom.py

```diff
@@ -40,3 +40,10 @@ def test_readDicom():
 
     fileInterface.fileWatcher.__del__()
     fileInterface.fileWatcher = None
+
+    # Test anonymization of sensitive patient fields
```

Comment:
For this test, maybe check before anonymization that at least some of the fields to be anonymized are present.